src/flask_container_scaffold/app_scaffold.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from flask import Flask
+
+from flask_container_scaffold.util import load_yaml
+
+
+class AppScaffold(object):
+
+    def __init__(self, app=None,
+                 name=__name__, config=None):
+        Flask.jinja_options = dict(Flask.jinja_options, trim_blocks=True,
+                                   lstrip_blocks=True)
+        self.app = app or Flask(name, instance_relative_config=True)
+        self.config = config
+        self._init_app(app)
+
+    def _init_app(self, app):
```

Comment:
`app` does not need to be passed in here since it's unused. This should normally throw a flake8 warning.